components/scrubber/scrubber.go

```diff
@@ -189,8 +236,7 @@ func (s *structScrubber) MapElem(m reflect.Value, k reflect.Value, v reflect.Val
 	}
 	if kind == reflect.String {
 		m.SetMapIndex(k, reflect.ValueOf(s.Parent.KeyValue(k.String(), v.String())))
```

Comment:
(note) ideally we should skip here as well to be sure that primitive won't double redact the value, but reflectwalk does not support skipping. Luckily testing showed that kind then is interface not string in primitive, so double redacting does not happen, and we have tests for it.

Reply:
Isn't that what `return reflectwalk.SkipEntry` does?